youtrack_api_get/issue_info.py

```diff
@@ -0,0 +1,49 @@
+import httpx
+import datetime
+
+state_id = "111-2"
+priority_id = "111-0"
+time_end_id = "117-0"
+
+
+def get_custom_field(youtrack_domain, issue_name, field_id, token):
+    url = "https://" \
+          + youtrack_domain \
+          + ".myjetbrains.com/youtrack/api/issues/" \
+          + issue_name \
+          + "/customFields/" \
+          + field_id
```

Comment:
Common url part move to class attribute.

Reply:
done